force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -355,18 +430,17 @@ public with sharing class ClassesObjectsMethods {
      * @return A list of new Person objects.
      */
     public static List<Person> createMultiplePersons(List<String> names, Integer age) {
-
+        List <Person> personList = new List <Person>();
         // Loop through each name in the provided list of names
+        for(String n : names){
             // Create a new Person object for each name
-            
+            Person person = new Person(n, age);
             // Add the new Person object to the list of Person objects
-        
-        
+            personList.add(person);
+        }
         // Return the list of Person objects
-        return null; // Replace null with the variable you used to store the result
+        return personList; // Replace null with the variable you used to store the result
     }
```

Comment:
In your loop I would add to the list directly and skip creating a variable on each iteration. 

```
  for (String name : names) {
            persons.add(new Person(name, age));
        }
        ```